build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -71,12 +71,6 @@ public static BuildTargetResult FinalizeBuild(BuildTargetContext c)
                 }
                 else
                 {
-                    // This is an old drop of latest so remove all old files to ensure a clean state
-                    AzurePublisherTool.ListBlobs($"{targetContainer}")
```

Comment:
Can we let the add up until your changes come in? We could search for files that match a regex but it's a little ugly.